### PR TITLE
Add Project.toml

### PR DESCRIPTION
**Pull request by martinholters**
We'll need this for future version registrations.

### CODE REVIEW

**Comment by galenlynch** [comment]
Thanks for doing this!

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Polynomials = "f27b6e38-b328-58d1-80ce-0feddd5e7a45"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+
+[compat]
+Polynomials = "≥ 0.1.0"
```
Comment (galenlynch) [inline Project.toml]: I can't find the relevant documentation in [Pkg's compatibility section](https://julialang.github.io/Pkg.jl/v1/compatibility/) for `≥`. Is it the same as a caret?

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name = "DSP"
+uuid = "717857b8-e6f2-59f4-9121-6e50c889abd2"
+
```
Comment (galenlynch) [inline Project.toml]: Don't we need a version?

File: Project.toml (hunk 1 above)
Comment (martinholters) [reply Project.toml]: https://julialang.github.io/Pkg.jl/v1/compatibility/#Inequality-specifiers-1

File: Project.toml (hunk 2 above)
Comment (martinholters) [reply Project.toml]: Good catch, we do! (Well, at least once we actually want to tag the next version, but no harm in setting it to the current version now.)

File: Project.toml (hunk 1 above)
Comment (galenlynch) [reply Project.toml]: Ah gotcha. Could we use the ASCII version? I haven't gotten around to integrating latex completions into my editor, and I feel like other people may be in a similar situation. Unicode symbols can be a real drag to work with if you're not using the right editor.

Comment (martinholters) [reply Project.toml]: Sure. This is what the automatic script produces.

**Comment by galenlynch** [comment]
Thanks again for doing this!

**Pull request merged**